src/components/Settings/PhoneAuthentication.vue

```diff
@@ -5,6 +5,7 @@
 			:visible="lightboxVisible"
 			@lightbox-closed="completeSetup"
 			:title="lightboxTitle"
+			:prevent-close="step === 3 ? 'true' : 'false'"
```

Comment:
I believe you could just do 
`:prevent-close="step === 3"`
here

Reply:
Thanks Ryan.